### PR TITLE
Adds Atomic documentation

### PR DESCRIPTION
**Pull request by chris-decker-volusion**
Reader-friendly version: https://github.com/volusion/element-documentation/blob/adds-atomic-documentation/references/styling-your-block-with-css/README.md

This documentation is mean to condense the following tickets:
- [V2-3569](https://volusion-product-team.atlassian.net/browse/V2-3569)
- [V2-3571](https://volusion-product-team.atlassian.net/browse/V2-3571)
- [V2-3503](https://volusion-product-team.atlassian.net/browse/V2-3503)

### CODE REVIEW

File: references/styling-your-block-with-css/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+## `joinClasses()`
+
+`joinClasses` is a helper function that is provided by the SDK and can be found in your block props. It is meant to join your classes together into a single string and contains some extra logic to ensure your class list is clean and free of unintended classes and whitespace.
+
+#### It will:
+
+-   Accept an infinite number of arguments and join them all together with a single space between each one.
+-   Remove any unnecessary whitespace that might be caused before, between, or after your final classes.
+-   Remove any invalid classes that might be output by bad logic (removes undefined, null, false, and additional whitespace), which is the biggest reason to use it over string literals.
+
```
Comment (divinentd) [inline references/styling-your-block-with-css/README.md]: you could add a 4th bullet here about conditional classes, like you showed below with `isRequirementMet`

File: references/styling-your-block-with-css/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+## Best Practices
+
+#### Avoid Inline Styling
+
+Sometimes inline styling is the best option for what you're trying to accomplish, but we recommend against using it when you can. Since you cannot use certain types of CSS with inline styling (media queries, pseudo selectors, etc) you end up creating yet another place that CSS needs to be tracked in your code. In the end, inline styling is fully supported, so do what works best.
+
+#### Avoid important tags
+
+In order to make a site Google AMP complaint, [you are unable to use `!important` tags in your CSS](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/style_pages/).
```
Comment (divinentd) [inline references/styling-your-block-with-css/README.md]:
```suggestion
In order to make a site Google AMP complaint, [you are unable to use `!important` qualifiers in your CSS](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/style_pages/).
```

File: references/styling-your-block-with-css/README.md (hunk 1 above)
Comment (chris-decker-volusion) [reply references/styling-your-block-with-css/README.md]: The 3rd bullet point covers this case as a string literal would include `false` for that line.

Comment (divinentd) [reply references/styling-your-block-with-css/README.md]: The third bullet reads like it is defensively protecting you from bad code, but what I'm getting at is that this function allows you to proactively add logic inside of your className using ternary or `&&` statements. Not knowing about this feature, there is a lot of Site Designer that has a bunch of class name logic up above the JSX that could have been inlined more simply.

Comment (chris-decker-volusion) [reply references/styling-your-block-with-css/README.md]: 661e95202f4d54ebb584da417337a6b282c5b601

File: references/styling-your-block-with-css/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+When the Atomic classes just aren't enough to meet your needs, you'll want to write custom styling with Aphrodite. You can view more information on styling your block with Aphrodite [here](../../how-to/style-a-block-with-aphrodite/README.md).
+
+## `joinClasses()`
+
+`joinClasses` is a helper function that is provided by the SDK and can be found in your block props. It is meant to join your classes together into a single string and contains some extra logic to ensure your class list is clean and free of unintended classes and whitespace.
+
+#### It will:
+
+-   Accept an infinite number of arguments and join them all together with a single space between each one.
```
Comment (chris-decker-volusion) [inline references/styling-your-block-with-css/README.md]:
```suggestion
-   Accept any number of arguments and join them all together with a single space between each one.
```

**Pull request merged**